packages/faustwp-core/src/version.ts

```diff
@@ -0,0 +1 @@
+export const LIB_VERSION = '0.2.1';
```

Comment:
This file should probably be git ignored. That way we won't need to increment it manually during a changesets release and since changesets does a build before release, this file will be created.